src/category_theory/limits/cones.lean

```diff
@@ -512,8 +512,7 @@ begin
   dsimp,
   rw [comp_id, H.map_comp, is_equivalence.fun_inv_map H, assoc, nat_iso.cancel_nat_iso_hom_left,
       assoc, is_equivalence.inv_fun_id_inv_comp],
-  apply comp_id,
-  -- annoyingly `dsimp, simp` leaves it as `c.π.app j ≫ 𝟙 _ = c.π.app j` instead of closing...
```

Comment:
What's wrong with using `comp_id` to close the goal here? I guess it could be faster?

Reply:
Nothing was wrong with `apply comp_id` as a proof. More it was that I was previously disappointed that `dsimp, simp` was mysteriously failing, and now happy to see that it was working properly.